src/App/__tests__/App.test.tsx

```diff
@@ -54,21 +54,10 @@ describe("App.tsx tests", () => {
     const footer = screen.getByRole("contentinfo");
     expect(footer).toBeInTheDocument();
 
-    expect(
-      screen.getByRole("link", {
-        name: marketLink,
-      })
-    ).toBeInTheDocument();
-    expect(
```

Comment:
Any tests related to navigating to `Leaderboard` or `Register` should be removed, as those links are no longer part of the header (thus causing the tests to fail).